docs/tables/aws_vpc_flow_log/index.md

```diff
@@ -195,17 +208,130 @@ partition "aws_vpc_flow_log" "my_logs_region" {
 }
 ```
 
-### Exclude rejected traffic logs
+### Collect logs for multiple regions
 
-Use the filter argument in your partition to exclude rejected traffic logs and reduce the size of local log storage.
+For all accounts, collect logs from us-east-1 and us-east-2.
 
 ```hcl
-partition "aws_vpc_flow_log" "my_logs_instance" {
-  filter = "action <> 'REJECT'"
-
-  source "aws_s3_bucket" {
+partition "aws_vpc_flow_log" "my_logs_region" {
+  source "aws_s3_bucket"  {
     connection  = connection.aws.vpc_logging
     bucket      = "vpc-flow-logs-bucket"
+    file_layout = `AWSLogs/(%{DATA:org_id}/)?%{NUMBER:account_id}/vpcflowlogs/(us-east-1|us-east-2)/%{YEAR:year}/%{MONTHNUM:month}/%{MONTHDAY:day}/(%{NUMBER:hour}/)?%{DATA}.log.gz`
+  }
+}
+```
+
+### Collect VPC flow logs exported from CloudWatch to an S3 bucket
+
+Collect VPC flow logs that were exported from CloudWatch to an S3 bucket, where the logs do not include a header line. Since the exported logs contain raw log lines only, a format block is defined to map the layout of each field.
+
+**Important Note**: The field, `export-timestamp`, is not part of the original log event. It is automatically added by AWS during the export process from CloudWatch to S3. This timestamp represents the time when the log was exported, not when the event occurred.
+
+```hcl
+format "aws_vpc_flow_log" "exported_log_format" {
+  layout = "export-timestamp instance-id interface-id pkt-srcaddr pkt-dstaddr pkt-src-aws-service az-id flow-direction start log-status packets protocol srcaddr dstaddr srcport end subnet-id"
+}
+```
+
+```hcl
+partition "aws_vpc_flow_log" "cw_log_group_logs" {
+  source "aws_s3_bucket" {
+    connection = connection.aws.logging_account
+    format     = format.aws_vpc_flow_log.exported_log_format
+    bucket     = "aws-vpc-flow-logs-123456789012-exported"
+    region     = "us-east-1"
+  }
+}
+```
+
+### Collect logs from a CloudWatch log group
```

Comment:
Please move this example up and review the order of other examples, we should make examples flow simple -> complex/edge

Reply:
Reordered the example configurations from simple to complex/edge cases.

@cbruno10, would it make sense to reorder the example configurations from simple to complex/edge cases and group them by source?